spec/features/bars/user_views_bars_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 feature "non-user views a list of bars" do
   scenario "sees all bars" do
-    new_bar = Bar.create(name: "Beantown Pub", description: "Cool bar, guy.")
-
+    user1 = User.create(email: "a4@gmail.com", password: "12345678")
```

Comment:
For the next iteration, follow the pattern that you have in `user_edits_bar_spec.rb`

```
let(:new_user) { User.create(email: "a3@gmail.com", password: "12345678") }
let(:bar1) { Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?", user: new_user) }
```